tasks/6to5.js

```diff
@@ -11,7 +11,12 @@ module.exports = function (grunt) {
 
 			var res = to5.transformFileSync(el.src[0], options);
 
-			grunt.file.write(el.dest, res.code);
+			var sourceMappingURL = '';
+			if (res.map) {
+				sourceMappingURL = grunt.util.linefeed + '//# sourceMappingURL=' + el.dest.split('/').pop() + '.map';
```

Comment:
> grunt.util.linefeed == \n, no?

No, most tools output in `\n`, so just `\n`.

> The el.dest could be a path, but you only want the filename part for the sourceMappingURL directive. Seems uglify is doing it that way, anyway?

I guess. Source Maps escape me. But use `path.basename()` instead then.